### ISSUE TITLE
συλλέγω

### ISSUE THREAD

**Issue by vgorman1** [alpheios-project/morpheus#43]
Lookup of the form συνέλεξε should go to συλλέγω.  Instead it goes to συν-λέγω1 and 3, with no definition or lexicon link.

![image](https://user-images.githubusercontent.com/10435794/102126054-2cfb5600-3e10-11eb-9b63-abf253c057c5.png)

**Comment by balmas** [comment]
@vgorman1 I have a question on this one

Is the morpheus parse of this as the compound of σύν and λέγω technically correct? I.e. is that equivalent to συλλέγω ? Or is συλλέγω distinct from that? Are all three correct?

**Comment by vgorman1** [comment]
Yes, it comes from συλλέγω, of which there is only one in the LSJ [there are three λεγω’s].

Vanessa B. Gorman
Professor of Ancient History
Department of History
619 Oldfather Hall
University of Nebraska-Lincoln
Lincoln, NE 68588-0327
https://vgorman1.github.io/

**Comment by balmas** [comment]
ok, so we still need an annotation specifying συλλέγω as a lemma of συνέλεξε

(with the fix for alpheios-project/alpheios-core#541, the user would be able to lookup the compound parts σύν and λέγω separately, but to fully handle that properly we need the support for compound lemmas described in alpheios-project/alpheios-core#135)
